jdbc/pom.xml

```diff
@@ -44,7 +44,13 @@
       <version>${project.version}</version>
       <scope>provided</scope>
     </dependency>
-
+	
+	<dependency>
```

Comment:
@vgmartinez I think this dependency doesn't need any longer. Could you please remove this?